### ISSUE TITLE
Convert all img elements to use srcset and responsive sizing

### ISSUE THREAD

**Issue by chrismejia** [SLB-Pizza/radio-pizza#122]
## What's the current situation?

Almost all `img` elements on site are using just one img size type, the default dictated by the img url coming in from Prismic CMS.

## What's the expected behavior?

All `img` elements have a default `src` attribute defined and a `srcset` attribute filled with responsive versions of images.

## What page/new page is this for?

**Page:** ALL
This has more to do with layout components than it does pages.

## File(s) involved, include line(s) if known

All **Site Elements, CMS Slices and Layout Helpers** that use `img` elements

---

## Prismic & Imgix Integration

The images coming from the GraphQL queries are URLs that have been [automatically compressed and resized by Imgix](https://user-guides.prismic.io/en/articles/3309829-image-optimization-imgix-integration).

HMBK CMS example: [Large Hadron Collider](https://images.prismic.io/hmbk-cms/a0964540-5c50-49f1-9d9e-f034dff29707_wallhaven-4g6k87.jpg?auto=compress,format&rect=0,0,3837,1200&w=1407&h=440)

URL: `https://images.prismic.io/hmbk-cms/a0964540-5c50-49f1-9d9e-f034dff29707_wallhaven-4g6k87.jpg?auto=compress,format&rect=0,0,3837,1200&w=1407&h=440`

## Basic HTML Implementation Example

https://codepen.io/snowballdigital/pen/KQGvYJ?editors=0010

## Related Issues

#116 - add responsive img sizes to CMS slices
#117 - add `srcset` attributes and sizings to frontend code

**Comment by chrismejia** [comment]
### Query Difference in Uncropped Photo; missing `compress`

![Imgur](https://images.prismic.io/hmbk-cms/a0964540-5c50-49f1-9d9e-f034dff29707_wallhaven-4g6k87.jpg?auto=compress,format "Large Hadron Collider")

**Prismic Image URL from GraphQL**
![Imgur](https://i.imgur.com/m6Vv1ki.png "Prismic Image URL from GraphQL")
- `"https://images.prismic.io/hmbk-cms/a0964540-5c50-49f1-9d9e-f034dff29707_wallhaven-4g6k87.jpg?auto=compress,format",`

vs.

**react-imgix processed Prismic URL**
![react-imgix processed Prismic URL, no compress, Large Hadron Collider](https://i.imgur.com/kWfSHD9.png "react-imgix processed Prismic URL, no compress, Large Hadron Collider")
- `"https://images.prismic.io/hmbk-cms/a0964540-5c50-49f1-9d9e-f034dff29707_wallhaven-4g6k87.jpg?auto=format&ixlib=react-9.0.2&h=1200&w=3840"`

### Query Difference in Cropped Photo? only `compress`

**Prismic Image URL from GraphQL, cropped image**

![Imgur](https://images.prismic.io/hmbk-cms/6dc7979b-4f92-41dd-90d4-03cf9dd1d402_pexels-tabitha-mort-775907.jpg?auto=compress,format&rect=320,0,1280,1280&w=1407&h=1407 "Bright lights cropped")

![Imgur](https://i.imgur.com/LLkLjJ1.png "Prismic Image URL from GraphQL")
- `"https://images.prismic.io/hmbk-cms/6dc7979b-4f92-41dd-90d4-03cf9dd1d402_pexels-tabitha-mort-775907.jpg?auto=compress,format&rect=320,0,1280,1280&w=1407&h=1407"`

vs.

**react-imgix processed Prismic URL**:
![Imgur](https://i.imgur.com/DFDJy2a.png "Prismic Image URL from GraphQL")
- `"https://images.prismic.io/hmbk-cms/6dc7979b-4f92-41dd-90d4-03cf9dd1d402_pexels-tabitha-mort-775907.jpg?auto=format&rect=320%2C0%2C1280%2C1280&w=1407&h=1407&ixlib=react-9.0.2&h=1407&w=1407&q=75&dpr=1 1x`

---

- [x] Check to see if _not having_ `compress` results in a larger file: _**IT DOES!**_

**Comment by chrismejia** [comment]
This issue would be workable if all images across all current Prismic types were the same.

The issue lies in changing the names/adding responsive sizes to images on a custom type. When you make a change, the images that **already exists** in a document of that Prismic type **DO NOT** update to create a new responsive size of that type. 

All the pre-change/addition responsive sizes remain, the newly modified size is `null`, a problem solvable by removing that image and re-selecting it in that document. This would need to be done to every image on every current document of every Prismic type, **AFTER** the responsive size names and dimension are unified.

The images that use `ResponsiveImage` are already set up and reaping those benefit.

TL;DR Too big a job for too little a gain, gonna mark this as `wontfix` and closing.
